### ISSUE TITLE
"File type invalid (1)" on upload to imgur with some albums

### ISSUE THREAD

**Issue by Jabster28** [phin05/discord-rich-presence-plex#42]
With certain albums, the upload will fail:
```py
[21-05-2022 07:28:34 PM] [DEBUG] [server/7D018] Uploading image
[21-05-2022 07:28:35 PM] [ERROR] An unexpected error occured while uploading an image
Traceback (most recent call last):
  File "/Users/me/discord-rich-presence-plex/services/imgur.py", line 15, in uploadImage
    raise Exception(data["data"]["error"])
```

I'm guessing this is because the album image isn't in the format that imgur wants, but I'm not sure how the upload system works.

**Comment by phineas05** [comment]
Add the below line in [services/imgur.py after line 7](https://github.com/phineas05/discord-rich-presence-plex/blob/da7d53d1d698642095559bd4be00930267e869cc/services/imgur.py#L7) and let me know what it outputs:
```python
logger.debug(requests.head(url).headers["content-type"])
```

**Comment by Jabster28** [comment]
```py
[24-05-2022 07:59:55 PM] [DEBUG] [server/8996D] Uploading image
[24-05-2022 07:59:55 PM] [DEBUG] image/jpeg
[24-05-2022 07:59:56 PM] [ERROR] An unexpected error occured while uploading an image
Traceback (most recent call last):
  File "/Users/me/discord-rich-presence-plex/services/imgur.py", line 16, in uploadImage
    raise Exception(data["data"]["error"])
Exception: {'code': 1003, 'message': 'File type invalid (1)', 'type': 'ImgurException', 'exception': []}
```

hmm, i'd assume it'd accept jpegs right?

**Comment by phin05** [comment]
This should be fixed in [v2.4.2](https://github.com/phin05/discord-rich-presence-plex/releases/tag/v2.4.2). Images supported by [Pillow](https://python-pillow.org/) will be converted to plain JPEG before being uploaded.

> hmm, i'd assume it'd accept jpegs right?

It should, but maybe the source image's content-type header is incorrect and the image is actually in a format not supported by Imgur.
